communication/moderation.md

```diff
@@ -1,85 +1,100 @@
-  - [Delete the post](https://support.google.com/groups/answer/1046523?hl=en) - 
-  - In some cases you might need to ban a user from the group, follow [these instructions](https://support.google.com/groups/answer/2646833?hl=en&ref_topic=2458761#) on how stop a member from being able to post to the group. 
+## Specific Guidelines
 
-For more technical help on how to use Google Groups, check the [Groups Help](https://support.google.com/groups/answer/2466386?hl=en&ref_topic=2458761) page.
+These guidelines are for tool-specific policies that don't fit under a general
+umbrella.
 
-### New users posting to a SIG/WG list
-New members who post to a group will automatically have their messages put in a queue and be sent the following message automatically: "Since you're a new subscriber you're in a moderation queue, sorry for the inconvenience, a moderator will check your message shortly."
+### Discuss guidelines
 
-Moderators will receive emails when messages are in this queue and will process them accordingly.
+- [Discuss guidelines](./discuss-guidelines.md)
 
-### Discuss
+### Mailing list guidelines
 
-- [Discuss Guidelines](./discuss-guidelines.md)
+- [Mailing List guidelines](./mailing-list-guidelines.md)
 
 ### Slack
 
-- [Slack Guidelines](./slack-guidelines.md)
+- [Slack guidelines](./slack-guidelines.md)
 
 ### Zoom 
 
-- [Zoom Guidelines](./zoom-guidelines.md)
+- [Zoom guidelines](./zoom-guidelines.md)
 
 
 ### References and Resources
 
-Thanks to the following projects for making their moderation guidelines public, allowing us to build on the shoulders of giants.
-Moderators are encouraged to learn how other projects moderate and learn from them in order to improve our guidelines:
+Thanks to the following projects for making their moderation guidelines public,
+allowing us to build on the shoulders of giants. Moderators are encouraged to
+learn how other projects moderate and learn from them in order to improve our
+guidelines:
 
 - Mozilla's [Forum Moderation Guidelines](https://support.mozilla.org/en-US/kb/moderation-guidelines)
 - OASIS [How to Moderate a Mailing List](https://www.oasis-open.org/khelp/kmlm/user_help/html/mailing_list_moderation.html)
 - Community Spark's [How to effectively moderate forums](http://www.communityspark.com/how-to-effectively-moderate-forums/)
 - [5 tips for more effective community moderation](https://www.socialmediatoday.com/social-business/5-tips-more-effective-community-moderation)
 - [8 Helpful Moderation Tips for Community Managers](https://sproutsocial.com/insights/tips-community-managers/)
 - [Setting Up Community Guidelines for Moderation](https://www.getopensocial.com/blog/community-management/setting-community-guidelines-moderation)
+
+[centralized list of administrators]: ./moderators.md#mailing-lists
+[CNCF Code of Conduct]: https://github.com/cncf/foundation/blob/master/code-of-conduct.md
```

Comment:
We should instead link to the Kubernetes code of conduct: https://github.com/kubernetes/community/blob/master/code-of-conduct.md since that's the source of truth for Kubernetes (we have it as a link pointing to CNCF's CoC  in k/community in case we decide to make changes to it in the future).

Reply:
SGTM. There should probably be another PR to update the other CoC links that still point directly to the CNCF:
* https://git.k8s.io/community/governance.md
* https://git.k8s.io/community/communication/README.md
* https://git.k8s.io/community/communication/discuss-guidelines.md
* https://git.k8s.io/community/communication/moderation.md
* https://git.k8s.io/community/communication/slack-guidelines.md
* https://git.k8s.io/community/communication/zoom-guidelines.md
* https://git.k8s.io/community/contributors/guide/README.md